requirements-win-cp27-cp34.txt

```diff
@@ -2,3 +2,4 @@
 cmake
 cython
 mingwpy
+pywin32

```

Comment:
File should be renamed to `requirements-win-cp27.txt`. `pywin32` isn't available for 3.4 and furthermore 3.4 is deprecated on March 2019 so supporting it seems far from necessary.